ci/run-tests.ps1

```diff
@@ -22,10 +23,11 @@ $output_dir_absolute = Force-ResolvePath $output_dir
 
 $cmd_args_list = @( `
     "`"$uproject_path_absolute`"", ` # We need some project to run tests in, but for unit tests the exact project shouldn't matter
+    "`"$test_repo_map`"", ` # The map to run tests in
     "-ExecCmds=`"Automation RunTests SpatialGDK; Quit`"", ` # Run all tests in the SpatialGDK group. See https://docs.unrealengine.com/en-US/Programming/Automation/index.html for docs on the automation system
     "-TestExit=`"Automation Test Queue Empty`"", ` # When to close the editor
     "-ReportOutputPath=`"$($output_dir_absolute)`"", ` # Output folder for test results. If it doesn't exist, gets created. If it does, all contents get deleted before new results get placed there.
-    "Log=`"$($log_file_name)`"", ` # Sets the name of the log file produced during this run. This file is saved in <Project folder>/Saved/Logs/. The lack of "-" is correct, -Log is a flag and doesn't set the file name
+    "-ABSLOG=`"$($log_file_path)`"", ` # Sets the name of the log file produced during this run. This file is saved in <Project folder>/Saved/Logs/. The lack of "-" is correct, -Log is a flag and doesn't set the file name
```

Comment:
What is ABSLOG? The comment will need an update if we keep this.

Reply:
I've updated the comment now. `Log` stores the log in `<Project folder>/Saved/Logs/`, `ABSLOG` allows to specify any arbitrary path for storing the log file. I'm dropping the log folder into the specially created `TestResults` directory, whose contents are included as artifacts on Buildkite.